### DISCUSSION TITLE
Changed datadir?

### DISCUSSION THREAD

**Discussion by andreasvikholm** [nextcloud/all-in-one#1641] [Questions] [answered]
Hi there,

I've installed nextcloud AIO docker container with 

```
sudo docker run \
--sig-proxy=false \
-e NEXTCLOUD_DATADIR=/mnt/ncdrive \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest-arm64
```
on my Raspberry Pi4 with an external drive connected and mounted at /mnt/ncdrive/ through fstab (that works, I can access it). Initially all worked very well. I moved my old datadir to the location and occ files:scan --all, and everything worked just fine fore some days.

Then I run out of usb-ports on my hardware, and after shutting everything down, I added an usb-hub, and connected the drive to it. If I understood it correctly, everything spun up fine at first, but at the same time a AIO Nextcloud upgrade where available, and I clicked update and... now the datadir seems to be lost by the AIO Nextcloud container.

The Nextcloud container log states

`2022-12-29 22:43:42.838077+01
(1 row)

Configuring Redis as session handler...
Setting php max children...
Appdata is not present. Did you maybe change the datadir after aio was first started?
See https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir
In the datadir was found:
total 8
drwxr-x---    2 www-data root          4096 Dec 29 22:43 .
drwxr-xr-x    1 root     root          4096 Dec  8 15:49 ..
nc: getaddrinfo: Name does not resolve
Waiting for database to start...
nc: getaddrinfo: Name does not resolve
Waiting for database to start...
nc: getaddrinfo: Name does not resolve
Waiting for database to start...
nc: getaddrinfo: Name does not resolve
Waiting for database to start...
`

Maybe it's obvious, but I'm fairly new to this world and my daily work has nothing to do with anything alike. 

Any ideas how to solve this? 

Happy new year!
Andreas

**Comment by szaimen** [comment, marked as answer]
Hi, is the drive still correctly mounted?

**Comment by 9at8** [comment]
@andreasvikholm Can you post the output of `lsblk -f` with the drive attached without a hub, and one where it's attached to the hub? Can you also post your fstab? `cat /etc/fstab`
